src/time_evolution/time_evolution.jl

```diff
@@ -1,9 +1,10 @@
 export TimeDependentOperatorSum
-export TimeEvolutionSol, TimeEvolutionMCSol
+export TimeEvolutionSol, TimeEvolutionMCSol, TimeEvolutionSSESol
 
 export liouvillian, liouvillian_floquet, liouvillian_generalized
 
 const DEFAULT_ODE_SOLVER_OPTIONS = (abstol = 1e-8, reltol = 1e-6, save_everystep = false, save_end = true)
+const DEFAULT_SDE_SOLVER_OPTIONS = (abstol = 1e-2, reltol = 1e-2, save_everystep = false, save_end = true)
```

Comment:
just a bit curious, is it okay to have such a low tolerance ?

Reply:
Yes, SDE problems are totally different from ODE ones. Even in DifferentialEquations.jl they put defaul values much higher than ODE ones.